ui/src/app/workflows/components/workflow-labels/workflow-labels.scss

```diff
@@ -0,0 +1,53 @@
+@import 'node_modules/argo-ui/src/styles/config';
+
+.wf-row-labels {
+  display: flex; 
+  flex-wrap: wrap;
+  align-items: center;
+  height: 100%;
+  padding: 1em 0;
+  $gutter: 0.5em;
+
+  &__action {
+    text-decoration: underline;
```

Comment:
In my opinion we should do this without the underline. We don't seem to use underlines to indicate links elsewhere in the UI. What do you think?

Reply:
I agree, sounds good. The underline looks a little dated IMO anyway. However, if it's just text I'm not sure that it's immediately obvious that it's clickable, so would adding a background to make it look like a button work?

Reply:
I think having the caret should be enough to tell users it's clickable. If we find that's not the case then we can always add something in the future